spherical_angle_bins: 100
egosphere_nodes: 20000
uncertainty_lower_bound: 10.0
